plugins/github-release-manager/src/components/Divider.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Divider as MaterialDivider } from '@material-ui/core';
+
+import { TEST_IDS } from '../test-helpers/test-ids';
+
+export const Divider = () => {
+  return (
+    <div style={{ margin: '20px 0' }} data-testid={TEST_IDS.components.divider}>
```

Comment:
Use the theme spacing for margins etc. in this case the simplest way should be `<Box margin={[2.5, 0]}>`, although the array there is a guess x)